plugins/woocommerce/tests/php/src/Internal/Caches/EntityVersionKeysCacheTest.php

```
@@ -0,0 +1,435 @@
+		$this->sut->delete_entity_version( 'product', true );
+	}
+
+	/**
+	 * @testdox Negative TTL from filter is converted to 0.
+	 */
+	public function test_negative_ttl_is_converted_to_zero() {
+		// phpcs:disable Generic.CodeAnalysis.UnusedFunctionParameter
+
+		add_filter(
+			'woocommerce_entity_version_key_ttl',
+			function ( $ttl, $entity_type, $entity_id ) {
```

PHPMD (2.15.0)

Avoid unused parameters such as '$entity_type'. (undefined)

(UnusedFormalParameter)